src/Datasource/Connection.php

```diff
@@ -61,7 +120,7 @@ protected function _normalizeConfig(array $config): array
                 throw new MissingConnectionException(['name' => $config['name']]);
             }
 
-            $config['driver'] = App::className($config['service'], 'Webservice/Driver');
+            $config['driver'] = App::className($config['service'], 'Webservice/Driver', 'Driver');
```

Comment:
```suggestion
            $config['driver'] = App::className($config['service'], 'Webservice/Driver');
```

Currently drivers don't have the `Driver` suffix (similar to database driver classes in the core) and there's no need to change it.

Reply:
Right, rolled that back